crates/voicevox_core_c_api/tests/e2e/float_assert.rs

```diff
@@ -0,0 +1,25 @@
+use indoc::indoc;
+use ndarray::Array;
+use ndarray_stats::DeviationExt as _;
+
+pub(crate) fn close_l1(test: &[f32], truth: &[f32], tol: f32) {
```

Comment:
assertを付けておくと意図がわかりやすいかもと思いました！
（あるいはRustで慣習的に付けられるtest時のprefixがあれば）
```suggestion
pub(crate) fn assert_close_l1(test: &[f32], truth: &[f32], tol: f32) {
```

Reply:
元がそういう名前なのでそれに従っています。
[`ndarray_linalg::assert::close_l1`](https://docs.rs/ndarray-linalg/latest/ndarray_linalg/assert/fn.close_l1.html)

`use`しないでフルの名前で用いることを想定されているんだと思いました (Rustにはこういうのがよくあります)。